lib/main_common.pm

```diff
@@ -66,6 +66,7 @@ our @EXPORT = qw(
   load_consoletests
   load_create_hdd_tests
   load_extra_tests
+  load_extra_tests_prepare
```

Comment:
```suggestion
```

Reply:
I had same thought at first look , but this function is actually used in `load_qam_publiccloud_tests` which is moved out of `main_common` so we do need to export it.

Reply:
I see now, thank you Anton